text/0814-stylelint.md

```diff
@@ -0,0 +1,180 @@
+---
+Stage: Accepted
+Start Date: 2022-04-19
+Release Date: Unreleased
+Release Versions:
+ember-cli: vX.Y.Z
+Relevant Team(s): Ember CLI
+RFC PR: https://github.com/emberjs/rfcs/pull/814
```

Comment:
Need to update front matter

Reply:
Can you explain what needs to be updated (or suggest the changes)? This comment isn't clear to me.

Reply:
The format of the YAML part at the top has changed. See https://raw.githubusercontent.com/emberjs/rfcs/master/0000-template.md

Reply:
This should be the frontmatter:
```
---
stage: accepted
start-date: 2022-04-19
release-date:
release-versions:
teams:
  - cli
prs:
  accepted: https://github.com/emberjs/rfcs/pull/814
project-link:
---
```